wg-serving/annual-report-2024.md

```diff
@@ -1,19 +1,109 @@
+
+The Gateway Inference Extension(GIE) has been developing rapidly, with our first release ([v0.1.0](https://github.com/kubernetes-sigs/gateway-api-inference-extension/releases/tag/v0.1.0)) recently published. We have already seen\[15%-60%\] reduction in output token latency when kv cache is close to saturation in this first release. 
+
+Development will continue rapidly, with a focus on: 
+
+* Productionalization  
+* Adoption of the latest developments (prefix caching as an example)  
+* And driving improvement/development in other areas of inference routing (multi-LoRA, disaggregated serving pools, etc)
+
+GIE has established adoption patterns for both the [Model Server](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/003-endpoint-picker-protocol#model-server-protocol), and the [Gateway](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/docs/proposals/003-endpoint-picker-protocol#proxy-protocol) interfaces. With regards to model servers, GIE already integrates with [vLLM](https://github.com/vllm-project/vllm) well, and will soon support [JetStream](https://github.com/AI-Hypercomputer/JetStream) as well, other model servers need only implement the protocol and will also cleanly integrate into GIE. As a part of sig-net, GIE seeks to build a strong partnership with the gateways in the space. Integration efforts from these orgs are already underway:
+
+* GKE  
+* [KGateway](https://kgateway.dev/)   
+* [Istio](https://istio.io/)
+
+GIE was developed on top of [ext-proc](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter) and [Envoy Proxy](https://www.envoyproxy.io/), and so any Proxy that can support ext-proc, can support the GIE protocol.
+
+The GIE team is looking forward to this upcoming year\! With many integrations upcoming and working with new partners (such as KServe) we look forward to where we are headed in 2025\.
+
+##### [Serving Catalog](https://github.com/kubernetes-sigs/wg-serving/tree/main/serving-catalog)
+
+Serving Catalog creates a repository of example K8s templates to deploy popular inference workloads. [Kustomized Blueprints - Serving Catalog](https://docs.google.com/document/d/1dOP0lIn-DK3tmq8gSvyK51J7CfKTz_Z1mqIAeYK8YS0/edit?tab=t.0#heading=h.48xpf8o6yz1j) described an approach for [LLM Serving Catalog](https://docs.google.com/document/d/1yXOhEXltc69_WFX90kMrHs3NcJGsD-U6-RejvK3XQtE/edit#heading=h.uxsw8fcrx2wp) using Kustomize overlays and components to provide a framework for extensible templates.
+
+The current support matrix is available [here](https://github.com/kubernetes-sigs/wg-serving/blob/main/serving-catalog/catalog.md), including support for:
+
+* [Singlehost Inference using Deployments](https://github.com/kubernetes-sigs/wg-serving/tree/main/serving-catalog/core/deployment) for vLLM and JetStream  
```

Comment:
single-host and multi-host?

Reply:
cc @jjk-g to double check

Reply:
I misread your comment previously. Updated.